packages/editor/src/components/editor-help/test/index.native.js

```diff
@@ -36,11 +36,11 @@ it( 'navigates back from help topic detail screen', async () => {
 	fireEvent.press( backButton[ backButton.length - 1 ] );
 
 	// Currently logs `act` warning due to https://github.com/callstack/react-native-testing-library/issues/379
-	await waitForElementToBeRemoved( () =>
-		screen.getByText(
-			'Each block has its own settings. To find them, tap on a block. Its settings will appear on the toolbar at the bottom of the screen.'
-		)
-	);
+	const text =
+		'Each block has its own settings. To find them, tap on a block. Its settings will appear on the toolbar at the bottom of the screen.';
+	await waitForElementToBeRemoved( () => screen.getByText( text ) );
+
+	expect( screen.queryByText( text ) ).toBeNull();
```

Comment:
I'm adding the assertion here to make it a full-featured test. We could use `.not.toBeInTheDocument()` but that's not supported by our custom React Native implementation yet, thus the `toBeNull()` usage.